tools/chromeos/gen_autosuspend_rules.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python2
```

Comment:
~~This must be run under python3. (meson is python3 only, and we certainly don't want to pull an obsolete stack just for this one script.)~~
So this is not called directly, but imported from `tools/make-autosuspend-rules.py`. It's misleading, but I guess keeping the script unchanged from upstream is more important. So this is OK.